scripts/on-prem-archive.sh

```diff
@@ -49,17 +49,29 @@ exists() {
 }
 
 s3_cp() {
-  aws s3 cp --acl=public-read "${1}" "${2}" >&2
+  hab pkg exec core/aws-cli aws s3 cp --acl=public-read "${1}" "${2}" >&2
 }
 
-check_tools() {
+install_tools() {
   for tool
   do
-    if ! exists "$tool"; then
-      echo "Please install $tool and run this script again."
-      exit 1
+    echo "Installing ${tool} from Habitat package."
+    if [[ "$tool" == "jq" ]]; then
+        HAB_AUTH_TOKEN="" hab pkg install core/jq-static
```

Comment:
I think maybe it would be better to check for root privileges and quit the script if they aren't present rather than obligating sudo here, simply because adding sudo calls will break the script in the studio.